File: Apps/React Native/master-store-rn/models/item.js
class Item {
  constructor(id, ownerId, title, imageUrl, description, price, date, email) {
    this.id = id;
    this.ownerId = ownerId;
    this.title = title;
    this.imageUrl = imageUrl;
    this.description = description;
    this.price = price;
    this.date = date;
    this.email = email;
  }
}

export default Item;

File: Apps/Flutter/master_store_f/lib/models/Item.dart
class Item {
  final String id;
  final String? ownerId;
  final String title;
  final String imageUrl;
  final String description;
  final int price;
  final String? date;
  final String? email;
  Item(this.id, this.ownerId, this.title, this.imageUrl, this.description,
      this.price, this.date, this.email);
}

class Cowboy {
  final String name;

  Cowboy(this.name);
}

List<Cowboy> demoContainerV = [Cowboy("name"), Cowboy("name"), Cowboy("name")];
